tests/CMakeLists.txt

```diff
@@ -1,5 +1,3 @@
-enable_testing()
-
```

Comment:
Using `include(CTest)` automatically enables testing, so there's no longer a need to enable it manually.